File: 3019-furthest-point-from-origin/furthest-point-from-origin.js

var furthestDistanceFromOrigin = function (moves) {
    let leftMove = 0
    let rightMove = 0
    let spacedMove = 0
    for (const move of moves) {
        if (move === 'L') leftMove++
        if (move === 'R') rightMove++
        if (move === '_') spacedMove++
    }
    return Math.abs(leftMove - rightMove) + spacedMove
};

File: 3019-furthest-point-from-origin/furthest-point-from-origin.cpp
class Solution {
public:
    int furthestDistanceFromOrigin(string moves) {
        int leftMove = 0;
        int rightMove = 0;
        int spacedMove = 0;
        for (char& move : moves) {
            if (move == 'L')
                leftMove++;
            if (move == 'R')
                rightMove++;
            if (move == '_')
                spacedMove++;
        }
        return abs(leftMove - rightMove) + spacedMove;
    }
};